google/cloud/security/common/gcp_api/appengine.py

```diff
@@ -39,11 +33,21 @@ class AppEngineClient(_base_client.BaseClient):
 
     API_NAME = 'appengine'
 
-    def __init__(self, credentials=None, version=None):
+    def __init__(self, global_configs, credentials=None, version=None):
+        """Initialize.
+
+        Args:
+            global_configs (dict): Global configurations.
+        """
         super(AppEngineClient, self).__init__(
-            credentials=credentials, api_name=self.API_NAME, version=version)
+            global_configs,
+            credentials=credentials,
+            api_name=self.API_NAME,
+            version=version)
+
         self.rate_limiter = RateLimiter(
-            FLAGS.max_appengine_api_calls_per_second, 1)
+            self.global_configs.get('max_appengine_api_calls_per_second'),
+            1)
```

Comment:
I think it was probably just easier to use "1" instead of creating a variable for this.  Fixing this now, as it's quick.